jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -0,0 +1,231 @@
+[#deprecated-system-environment-variables]
+== Deprecated system environment variables
+
+There are a number of built-in environment variables that are not available in GitHub-based projects for accounts authenticated through GitHub Apsps. VCS support for each environment variable is indicated in the xref:variables#built-in-environment-variables[Built-in environment variables] table on the Project values and variables page. If your pipelines need these environment variables, we recommend you use suitable replacements from the available xref:pipeline-variables#[pipeline values].
+
+[#coming-soon]
+== Coming soon
+
+The following sections are features of CircleCI which are not currently fully supported. These features are planned for future releases.
+
+[#manual-trigger-pipeline-option]
+=== Manual trigger pipeline option
+The ability to manually trigger a pipeline from the web app is not currently supported for GitHub Apps projects.
+
+[#in-app-config-editor]
+=== In-app config editor
+The in-app config editor is currently **only** available for GitHub App accounts during project creation.
+
+[#account-integrations]
+=== Account integrations
+
+There is currently no method to manage the connection with GitHub outside of the project setup, trigger, and configuration settings. CircleCI is working on enabling users to manage their users’ GitHub identity as part of their user profile's account integration settings.
+
+[#scheudled-pipelines]
+=== Scheduled pipelines
+
+The ability to xref:scheduled-pipelines#[schedule pipelines] is not currently supported for GitHub Apps projects. This feature is planned for a future release.
+
+[#auto-cancel-redundant-workflows]
+=== Auto-cancel redundant workflows
+
+Auto-cancel redundant workflows is not currently supported. It is often used to remove noise from the pipeline page and lower the time to feedback for a commit. Refer to the xref:skip-build#auto-cancelling[Skip or cancel jobs and workflows] page for more details.
+
+[#passing-secrets-to-forked-pull-requests]
+=== Passing secrets to forked pull requests
+
+Passing secrets to forked pull requests is not currently supported.
+
+[#stop-building]
+=== Stop building
+
+GitHub App integrations do not currently support the **Stop Building** option that can normally be found in **Project settings**.
+
+The recommendation is to either:
+
+* Suspend your installation. This would stop sending all events to CircleCI, so all builds will stop. This option is available in GitHub **Organization settings** under the **GitHub Apps** menu option.
+* Stop a single project from sending events to CircleCI. This option is available in GitHub **Organization settings** under the **GitHub Apps** menu option. Under **Repository access**, select **Only select repositories** and deselect the repository you want to stop building.
+
+[#ssh-rerun]
+=== SSH rerun
+
+SSH reruns are not currently supported for GitHub App projects. This feature is in development and will be available soon.
```

Comment:
I think we should use similar language as we do for auto-cancel redundant builds and scheduled pipelines. It's unclear when we'll be able to prioritize working on SSH reruns. I would just say "will be available in a future release".